core/src/apps/bitcoin/scripts_decred.py

```diff
@@ -3,7 +3,9 @@
 from trezor import utils
 from trezor.crypto import base58
 from trezor.crypto.base58 import blake256d_32
+from trezor.enums import DecredStakingSpendType
 from trezor.wire import DataError
+from micropython import const
```

Comment:
should be moved before `from typing`, as the very first line. like here: https://github.com/trezor/trezor-firmware/blob/matejcik/coins-t3b1/core/src/apps/binance/helpers.py#L1